examples/temporal/main.js

```diff
@@ -0,0 +1,571 @@
+                    stateset.setAttributeAndModes( new osg.Depth( 'DISABLE' ) );
+
+                    _ComposerdebugNode.addChild( quad );
+
+                    if ( optionsDebug.horizontal ) xOffset += optionsDebug.w + 2;
+                    else yOffset += optionsDebug.h + 2;
+                }
+            }
+        },
+
+
+        createTextureRTT: function ( name, filter, type ) {
+            var texture = new osg.Texture();
+            texture.setInternalFormatType( type );
+            texture.setTextureSize( this._canvas.width, this._canvas.height );
+
+            texture.setInternalFormat( osg.Texture.RGBA );
+            texture.setMinFilter( filter );
+            texture.setMagFilter( filter );
+            texture.setName( name );
+            return texture;
+        },
+
+        createCameraRTT: function ( texture, is3D ) {
+            var camera = new osg.Camera();
+            camera.setName( is3D ? 'MainCamera' : 'composer2D' );
+            camera.setViewport( new osg.Viewport( 0, 0, this._canvas.width, this._canvas.height ) );
+
+            camera.setRenderOrder( osg.Camera.PRE_RENDER, 0 );
+            camera.attachTexture( osg.FrameBufferObject.COLOR_ATTACHMENT0, texture, 0 );
+
+            //
+            camera.setReferenceFrame( osg.Transform.ABSOLUTE_RF );
+
+            if ( is3D ) {
+                camera.attachRenderBuffer( osg.FrameBufferObject.DEPTH_ATTACHMENT, osg.FrameBufferObject.DEPTH_COMPONENT16 );
+                camera.setClearColor( osg.Vec4.create( [ 0.0, 0.0, 0.1, 1.0 ] ) );
+            } else {
+
+                camera.setClearMask( 0 );
+
+            }
+            return camera;
+        },
+        ///// END UTILS
+
+        // halton[0-1] : jitter offset
+        // halton[2] : 0 disable; 1: supersample, 2: motion blur
+        // halton[3] : Frame Number since start of amortized supersample of static frame
+
+        //http://www.ben-peck.com/articles/halton/
+        haltonFunc: function ( index, base ) {
+            var result = 0.0;
+            var f = 1.0 / base;
+            var i = index;
+            while ( i > 0 ) {
+                result = result + f * ( i % base );
+                i = Math.floor( i / base );
+                f = f / base;
+            }
+            return result;
+        },
+
+        updateUniforms: function () {
+
+            if ( !this._halton ) return;
+            // update frame num to know how to amortize
+            this._frameNum++;
+
+            // jitter the projection matrix
+            this._halton[ 0 ] = this.haltonFunc( this._frameNum - 1, 4 ) - 0.5;
+            this._halton[ 1 ] = this.haltonFunc( this._frameNum - 1, 3 ) - 0.5;
+
+            this._halton[ 3 ] = this._frameNum;
+
+            this._uniforms.halton.dirty();
+
+        },
+
+
+
+
+        rotateNode: function ( nv ) {
+            var t = nv.getFrameStamp().getSimulationTime();
+            var dt = t - this._rotate._lastUpdate;
```

Comment:
this._rotate._lastUpdate should not be allowed. if you should alawys use an accessors if access from external the class. It's acceptable because of it's an example but should always be avoided